server/pypi/build-wheel.py

```diff
@@ -617,7 +617,8 @@ def env_vars(self):
         })
 
         for var in self.meta["build"]["script_env"]:
-            key, value = var.split("=")
+            key, value = var.split("=",1)
```

Comment:
This was failing because of finding 
IGRAPH_CMAKE_EXTRA_ARGS=-DF2C_EXTERNAL_ARITH_HEADER=../../../../../../../android64_x86_64_arm64.h -DIEEE754_DOUBLE_ENDIANNESS_MATCHES=ON

Multiple "=", now will only take the first segment as key and the rest as value as is